src/app/pages/data-protection/cloudsync/cloudsync-form/cloudsync-form.component.ts

```diff
@@ -40,6 +41,12 @@ import {
 } from 'app/services';
 import { ModalService } from 'app/services/modal.service';
 
+type CloudSyncFormTask = Overwrite<CloudSyncTask, {
```

Comment:
`Overwrite` usage looks more like a hack. Wouldn't it be better to declare a class?

Reply:
Agree that it's a hack, but it's a hack because somebody put a different type in field. We will refactor this component at some point and this will go away.